rust/src/lib/ip.rs

```diff
@@ -146,6 +148,14 @@ pub struct InterfaceIpv4 {
     /// which result the IP addresses after first one holding the `secondary`
     /// flag.
     pub addresses: Option<Vec<InterfaceIpAddr>>,
+    /// Whether per‐interface IPv4 sysctl forwarding is enabled.
+    /// When applying with `Some(true)`, nmstate will set
+    /// `net.ipv4.conf.<interface>.forward = 1` (enable forwarding).
+    /// When applying with `Some(false)`, nmstate will set
+    /// `net.ipv4.conf.<interface>.forward = 0` (disable forwarding).
```

Comment:
No need to mention internal implementation in API document. Just state `enable IPv4 forwarding` is enough.